scripts/GMT_age.sh

```diff
@@ -2,7 +2,7 @@
 # Makes crustal age map for docs/earth-age.rst
 gmt begin GMT_age png
 	gmt set MAP_TICK_LENGTH 2p MAP_ANNOT_OFFSET 2p MAP_FRAME_PEN 0.5p GMT_DATA_SERVER candidate
-	gmt grdimage @earth_age_06m -JQ180/6i -C@earth_age.cpt -B0 --MAP_FRAME_PEN=1p
```

Comment:
Use a lower-resolution grid to have a smaller image size?

Reply:
Fine by me. If I use `@earth_age` instead, them the 10m resolution is used and the png is 112K. 

Paul?

Reply:
10m is fine with me